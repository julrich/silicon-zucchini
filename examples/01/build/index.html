<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>WELCOME!!</title>
</head>
<body>
<article>
  <h1>
    WELCOME!!
  </h1>
</article>

<script src="http://localhost:35729/livereload.js"></script>

</body>
</html>