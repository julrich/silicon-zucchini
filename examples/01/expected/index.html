<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>WELCOME!!</title>
</head>
<body>

<nav>
  <ul>
  
    <li>
      <a href="/artikel/">
        Articles
      </a>
    </li>
  
    <li>
      <a href="/artikel/welcome-to-zombocom/">
        Welcome to ZOMBO.COM
      </a>
    </li>
  
    <li>
      <a href="/">
        WELCOME!!
      </a>
    </li>
  
    <li>
      <a href="/uber/">
        Ãœber
      </a>
    </li>
  
  </ul>
</nav>

<h1>WELCOME!!</h1>
<div class="content">
  
</div>


</body>
</html>